pyro/contrib/gp/likelihoods/gaussian.py

```diff
@@ -12,9 +12,13 @@
 
 class Gaussian(Likelihood):
     """
-    Implementation of Gaussian likelihood.
+    Implementation of Gaussian likelihood, which is used for regression problems.
```

Comment:
Add a note about conjugacy properties or say it's not exploited?

Reply:
Could you elaborate more for me about conjugacy properties and not exploited?